### PR TITLE
JVM Logs Integration

### PR DESCRIPTION
**Pull request by gr0**
This PR introduces the JVM Logs integration to our docs.

### CODE REVIEW

File: docs/integration/jvm-logs.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+title: JVM Logs Integration
+description: Sematext JVM Logs integration allows you to check grabeg collector logs, times the applications threads were stopped, memory regions size and much more.
```
Comment (otisg) [inline docs/integration/jvm-logs.md:2]: typo

File: docs/integration/jvm-logs.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+title: JVM Logs Integration
```
Comment (otisg) [inline docs/integration/jvm-logs.md:1]: It is just "JVM Logs" or specifically "JVM Garbage Collection Logs"?

File: docs/integration/jvm-logs.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+description: Sematext JVM Logs integration allows you to check grabeg collector logs, times the applications threads were stopped, memory regions size and much more.
+
+To make use of the Sematext JVM Logs integration, you'll need to install the Sematext Agent and configure it to ship garbage collector logs to your JVM Logs App via the [Logs Discovery](../logs/discovery/intro.md).
```
Comment (otisg) [inline docs/integration/jvm-logs.md:4]: Sematext Agent could be linked?

File: docs/integration/jvm-logs.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+<img
+  class="content-modal-image"
+  alt="Solr G1 GC Report"
+  src="../../images/agents/jvm_logs_g1gc.png"
+  title="Solr G1 GC Report"
+/>
+
+Other built-in reports include:
+
+- **Errors**: breakdown of errors comming from your logs
```
Comment (otisg) [inline docs/integration/jvm-logs.md:30]: coming

File: docs/integration/jvm-logs.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- **G1GC**: report dedicated to the garbge first garbage collector and its metrics
```
Comment (otisg) [inline docs/integration/jvm-logs.md:31]: typo

File: docs/integration/jvm-logs.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  src="../../images/agents/jvm_logs_g1gc.png"
+  title="Solr G1 GC Report"
+/>
+
+Other built-in reports include:
+
+- **Errors**: breakdown of errors comming from your logs
+- **G1GC**: report dedicated to the garbge first garbage collector and its metrics 
+- **Meta**: meta and class regions dedicated report
+- **Suvrivor Stats**: information regarding the survivor region
```
Comment (otisg) [inline docs/integration/jvm-logs.md:33]: typo

**Review by otisg — CHANGES_REQUESTED** [review]
We should mention alerts?
Show split screen with a metric spike and a GC spike?

File: docs/integration/jvm-logs.md (hunk 1 above)
Comment (gr0) [reply docs/integration/jvm-logs.md:2]: Thanks, adjusted.

File: docs/integration/jvm-logs.md (hunk 2 above)
Comment (gr0) [reply docs/integration/jvm-logs.md:1]: Adjusted, thanks for pointing that out.

File: docs/integration/jvm-logs.md (hunk 3 above)
Comment (gr0) [reply docs/integration/jvm-logs.md:4]: Done.

File: docs/integration/jvm-logs.md (hunk 4 above)
Comment (gr0) [reply docs/integration/jvm-logs.md:30]: Thanks, adjusted.

File: docs/integration/jvm-logs.md (hunk 6 above)
Comment (gr0) [reply docs/integration/jvm-logs.md:33]: Thanks, adjusted.

**Comment by gr0** [comment]
The JVM GC Logs integration logs looks as follows:

gc_logs_docs

**Comment by otisg** [comment]
+1 for the Alerts part.  But that Split Screen is not about Alerts.  Shouldn't we add something a la https://sematext.com/docs/integration/solr/#correlation-with-logs ?

**Comment by gr0** [comment]
Adjusted. Looks like this right now:

docs

**Review by otisg — APPROVED** [review]
LGTM!

**Comment by gr0** [comment]
Thanks :) I'll wait with merge once we have everything ready.

**Pull request merged**